stingray/largememory.py

```diff
@@ -565,6 +564,7 @@ def retreiveData(data_type, dir_name, path=os.getcwd(), chunk_data=False, chunk_
         else:
             return _retrieveDataLC(data_path, raw=raw)
 
+    # REVIEW: Check need for creating seperate fits, retrieve function for extensibility and due to different data
```

Comment:
Do we need a special `retrieveFITS` function? Currently, the `retrieveEventList` function is used for both _EventList_ and _FITS_ but the disadvantage of this is that the metadata - tstart, tstop from FITS isn't retrieved.

I feel a special function for retrieving FITS data will help with extensibility. What are your views?